assets/js/blocks/checkout/inner-blocks/checkout-shipping-methods-block/edit.tsx

```diff
@@ -119,6 +120,7 @@ export const Edit = ( {
 						</ExternalLink>
 					</PanelBody>
 				) }
+				<CheckoutSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 60).